dm/dumpling/dumpling.go

```diff
@@ -164,7 +164,9 @@ func (m *Dumpling) Process(ctx context.Context, pr chan pb.ProcessResult) {
 
 	newCtx, cancel := context.WithCancel(ctx)
 	var dumpling *export.Dumper
-
+	failpoint.Inject("longDumpProcess", func() {
+		m.dumpConfig.Rows = 0
```

Comment:
why we need to set it?

Reply:
forbid concurrent dumping. But seems redundant here...it's 0 by default.

Reply:
20edb4b